devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -122,17 +117,93 @@ enum rpcCommand
     resetCalibration,
 };
 
+enum TargetType
+{
+    pose,
+    poseAndVelocity,
+    position,
+    positionAndVelocity,
+    orientation,
+    orientationAndVelocity
+};
+
+static std::unordered_map<std::string,TargetType> const stringToTargetType = { {"pose",TargetType::pose},
+                                                                               {"poseAndVelocity",TargetType::poseAndVelocity},
+                                                                               {"position",TargetType::position},
+                                                                               {"positionAndVelocity",TargetType::positionAndVelocity},
+                                                                               {"orientation",TargetType::orientation},
+                                                                               {"orientationAndVelocity",TargetType::orientationAndVelocity}};
+
+class WearableSensorTarget
```

Comment:
This file is pretty huge. Consider moving the additional classes to separate files